src/app/tree-grid/tree-grid-row-edit-style/tree-grid-row-edit-style.component.html

```diff
@@ -0,0 +1,26 @@
+<div class="grid__wrapper">
+    <igx-tree-grid #treeGrid [data]="data" primaryKey="ID" foreignKey="ParentID" width="100%" height="500px"
+        rowEditable=true rowSelectable=true columnHiding=true>
+        <igx-column *ngFor="let c of columns" [editable]="c.editable" [field]="c.field" [dataType]="c.dataType"
+            [header]="c.label" [movable]="c.movable" [resizable]="c.resizable" [sortable]="c.sortable"
+            [filterable]="c.filterable">
+        </igx-column>
+        <ng-template igxRowEditText let-rowChangesCount>
+            <span class="changes"> Changes: {{rowChangesCount}}</span>
+        </ng-template>
+        <ng-template igxRowEditActions let-endRowEdit>
```

Comment:
The icon buttons from the template do not apply for the igxTree sample and the default ones are actually rendered. Can you please take a look what could be causing it?

Reply:
Thanks for catching this! I've logged an issue in the [main repo](https://github.com/IgniteUI/igniteui-angular/issues/5431)